src/binary/Yast.cc

```diff
@@ -521,6 +521,10 @@ static VALUE ui_set_component(VALUE self, VALUE name)
 
 static void init_ui()
 {
+  // init_ui is needed only if run via rspec, if y2base is used ENV is initialized
```

Comment:
in fact, here y2base stands for both :)
what comment want to say, is that code is used only for test purpose. Real code initialize it on its own, but as init_ui is called early, we need to skip it for production code.